sockeye/inference.py

```diff
@@ -303,6 +323,10 @@ def make_input_from_dict(sentence_id: SentenceId,
                 logger.error("There is mismatch in source factors %d and prefix factors %d", len(factors), len(source_prefix_factors))
                 return _bad_input(sentence_id, reason=str(input_dict))
 
+        target_prefix_tokens = input_dict.get(C.JSON_TARGET_PREFIX_KEY)
+        target_prefix_tokens = list(utils.get_tokens(target_prefix_tokens)) if target_prefix_tokens else None
+        keep_target_prefix_key = input_dict.get(C.JSON_KEEP_TARGET_PREFIX_KEY)
+        keep_target_prefix_key = False if (keep_target_prefix_key is not None and keep_target_prefix_key == "False") else True
```

Comment:
would the string value of `keep_target_prefix_key` be `"False"` or `"false"` or `False` ? I think we should follow JSON convention, see my comment above.

Reply:
Done. Thanks!